app/screens/Observations/map.js

```diff
@@ -100,8 +103,38 @@ class ObservationMapScreen extends Component {
     this.refs["menuInstance"].setVelocity({ x: 3000 });
   }
 
-  componentDidMount() {
-    console.log("map", this._map);
+  getFeatures() {
```

Comment:
I was expecting this to return something rather than updating the state, so maybe a name like `prepareAnnotations` makes more sense?